.sync.yml

```diff
@@ -13,6 +13,7 @@ Gemfile:
         version: '~>2.14.1'
       - gem: puppet-lint
       - gem: puppetlabs_spec_helper
+        version: '~>0.10.3'
```

Comment:
There was no dependency issues moving up to the latest, and knowing the latest has the best of everything, just adding it as that version. Basically the logic mentioned in the commit itself.